core/templates/dev/head/pages/skill-editor-page/navbar/skill-editor-navbar.directive.html

```diff
@@ -86,7 +85,14 @@
   </li>
 </ul>
 <style>
-  skill-editor-navbar .save-draft-button-padding {
-    padding-right: 120px;
+  .btn-success:hover {
+    background-color: #218838 ;
```

Comment:
Extra space before semicolon, here and below.

Reply:
Done Thanks!